docs/protocol-methods.rst

```diff
@@ -498,6 +568,119 @@ Unsubscribe from a script hash, preventing future notifications if its :ref:`sta
   Note that :const:`False` might be returned even for something subscribed to earlier,
   because the server can drop subscriptions in rare circumstances.
 
+blockchain.outpoint.subscribe
+=============================
+
+Subscribe to a transaction outpoint (TXO), to get notifications about its status.
+A status involves up to two transactions: the funding transaction that creates
+the TXO (as one of its outputs), and the spending transaction that uses it
+as an input (spends it).
```

Comment:
Is it worth spelling out the client should get unsubscribed in case of lost connection?

Also I suggest: The server MAY automatically unsubscribe an outpoint if the spending transaction is 100 or more blocks deep. The server MAY ignore subscription request if the spending transaction is 100 or more blocks deep but it still must send at least one full response.

Rationale: 100 blocks-deep reorgs are considered impossible (coinbase rule) and thus there will be no more notifications. Cleaning up the connections may avoid problems if the client fails to unsubscribe.

Reply:
Makes sense, thanks!